Formula/cassowary.rb

```diff
@@ -2,8 +2,9 @@ class Cassowary < Formula
   desc "Modern cross-platform HTTP load-testing tool written in Go"
   homepage "https://github.com/rogerwelin/cassowary"
   url "https://github.com/rogerwelin/cassowary/archive/v0.14.0.tar.gz"
-  sha256 "385232478b8552d56429fbe2584950bfbe42e3b611919a31075366a143aae9a9"
+  sha256 "04fbf1e563378bc37a5ad44e69a3b100647406ea2debae2d38d3c406399628fa"
```

Comment:
The short URL is hitting some duplicate, which is causing it to not resolve actual tarball URL. May be due to Git tag and branch with same name:
```console
❯ curl https://github.com/rogerwelin/cassowary/archive/v0.14.0.tar.gz -L
the given path has multiple possibilities: #<Git::Ref:0x00007fd87706cf50>, #<Git::Ref:0x00007fd877072428>
```

We will need to use longer URL (https://github.com/rogerwelin/cassowary/archive/refs/tags/v0.14.0.tar.gz)